src/java.base/share/classes/java/lang/classfile/attribute/SyntheticAttribute.java

```diff
@@ -34,7 +34,7 @@
 import jdk.internal.javac.PreviewFeature;
 
 /**
- * Models the {@code Synthetic} attribute {@jvms 4.7.8}, which can appear on
+ * Models the {@code Synthetic} attribute JVMS {@jvms 4.7.8}, which can appear on
```

Comment:
These occurrences should be `(JVMS {@jvms x.x.x})`.

Reply:
I agree that structure would be better and more consistent with `@jmvs` usage elsewhere in core libs.